src/uu/numfmt/src/numfmt.rs

```diff
@@ -201,46 +203,86 @@ fn remove_suffix(i: f64, s: Option<Suffix>, u: &Unit) -> Result<f64> {
 
 fn transform_from(s: &str, opts: &Transform) -> Result<f64> {
     let (i, suffix) = parse_suffix(s)?;
-    remove_suffix(i, suffix, &opts.unit).map(|n| n.round())
+
+    remove_suffix(i, suffix, &opts.unit).map(|n| match n < 0.0 {
+        true => n.abs().ceil() * -1.0,
+        false => n.ceil(),
```

Comment:
Stuff like this should just use an `if` expression.  This comment applies pretty much everywhere in this change except for the `match` expressions dealing with `enum`s.

Reply:
Great, thanks! 

I'm still trying to get the feel for when match expressions make sense. For this spot, are you suggesting something like this?

```
    remove_suffix(i, suffix, &opts.unit).map(|n| if n < 0.0 { -n.abs().ceil() } else { n.ceil() })
```